sse2neon.h

```diff
@@ -9623,18 +9623,19 @@ FORCE_INLINE __m128i _mm_aesdec_si128(__m128i a, __m128i RoundKey)
     return vreinterpretq_m128i_u8(w) ^ RoundKey;
 
 #else /* ARMv7-A NEON implementation */
-/* FIXME: optimized for NEON */
-#define XT(x) (((x) << 1) ^ ((((x) >> 7) & 1) * 0x1b))
-#define MULTIPLY(x, y)                                                     \
-    (((y & 1) * x) ^ ((y >> 1 & 1) * XT(x)) ^ ((y >> 2 & 1) * XT(XT(x))) ^ \
-     ((y >> 3 & 1) * XT(XT(XT(x)))) ^ ((y >> 4 & 1) * XT(XT(XT(XT(x))))))
-
+    /* FIXME: optimized for NEON */
     uint8_t i, e, f, g, h, v[4][4];
     uint8_t *_a = (uint8_t *) &a;
     for (i = 0; i < 16; ++i) {
         v[((i / 4) + (i % 4)) % 4][i % 4] = SSE2NEON_rsbox[_a[i]];
     }
 
+#define XT(x) (((x) << 1) ^ ((((x) >> 7) & 1) * 0x1b))
```

Comment:
There are potentially conflicts when you define macro `XT` and `MULTIPLY`. You have to add `SSE2NEON_` prefix.

Reply:
By the way, you should modify the existing usage of `XT` and `MULTIPLY` as well. I forgot to raise the concerns.